src/diffusers/quantizers/bitsandbytes/bnb_quantizer.py

```diff
@@ -204,7 +204,10 @@ def create_quantized_param(
 
         module._parameters[tensor_name] = new_value
 
-    def check_quantized_param_shape(self, param_name, current_param_shape, loaded_param_shape):
+    def check_quantized_param_shape(self, param_name, current_param, loaded_param):
```

Comment:
GGUF needs to access the tensor quant type to run a shape check. So this needs to change from passing in shapes to passing in params directly.

Reply:
Why not add this method to the `gguf_quantizer.py` file instead of modifying this? This would be a breaking change no?

Reply:
I see you're already adding this to the GGUF quantizer class. So, maybe okay to not modify this?